src/components/atoms/Logo/Logo.tsx

```diff
@@ -38,11 +38,11 @@ export const Logo: FC<LogoProps> = ({
 }
 
 // S T Y L E S
-const StyledLogoWrapper = styled.span`
-  --logo-height: 2rem;
+const StyledLogoWrapper = styled.div`
+  --logo-height: 40px;
```

Comment:
Shouldn't all height measurements be in 'rem' or 'em'?

Reply:
Ok, this needs to be discussed with MN

Reply:
will change that after the spacing session with moe